### ISSUE TITLE
OpenStack Builder Plugin

### ISSUE THREAD

**Issue by mkandes** [singularityhub/sregistry#274]
Hi Vanessa,

It looks like there is no OpenStack Builder plugin. Is that correct? If so, I might have some time to dedicate to this feature. I support quite a few containers that are too "big" to really build on SingularityHub and/or Sylabs Library within the resource limits. And I'm sort of over building these containers on my laptop. 

Anyhow, if there isn't any OpenStack support yet. Let me know. I'd be interested in prototyping such a plugin to help out the project.

Marty

**Comment by vsoch** [comment]
Yes, yes, and yes! You can use any of the other plugins as a template - and ping me with any questions! Make sure that it’s modular (like the others) so that the user can enable it via the Dockerfile and PLUGINS_INSTALLED in the settings folder.

**Comment by mkandes** [comment]
Will do. I've got a few projects to clear off my plate first. But I'll definitely take a stab at this as soon as I free up a bit of time.

**Comment by vsoch** [comment]
@mkandes any updates, do you need any help? I'd really like to learn about making such a plugin, if you eventually can find some time. I know that the entire world is kind of weird and different now, so please take all the time that you need (and make sure to ping me if I can help!)

**Comment by mkandes** [comment]
@vsoch: No updates yet. I'm still trying to clear off these other projects first. But yes, I think I might have some "free time" over the next few months to get a prototype of this up and running. I'll definitely be pinging you with any questions once I dig in.
